less/variables.less

```diff
@@ -24,3 +24,4 @@
 @border-real: #dee2e6;
 
 // Mobile var
+@phone: ~"(max-width: 500px)";
```

Comment:
Mobile Design - Sean, you did an amazing job here too! 

 Create a @phone variable that contains a max-width: 500px media query string. Use the @phone variable for all your nested mobile styling. - DONE!

 Review the provided home mobile design file. Match your mobile styling the best you can using the design file. - WELL DONE!